bin/run_unit_tests.sh

```diff
@@ -13,15 +13,13 @@ REPORT_COVERAGE=${CI:-""}
 if [ "$REPORT_COVERAGE" = "true" ]; then
     make install-goveralls
 fi
-# run app unit tests
-make test
-# run monitor unit tests
-make -C tools/autograph-monitor test
 
 if [ "$RACE_TEST" = "1" ]; then
     make race
     make -C tools/autograph-monitor race
```

Comment:
Do we know why `autograph-monitor` is treated specially rather than being listed in `PACKAGE_NAMES`?

Reply:
Looked into this, and I'm not sure. The tooling may have been worse back in the day and not allowed you to run `go test` on two different `package main`s, or the original developers expected to have to do more in the tools/autograph-monitor/Makefile to make the package testable.

So, to prove that out, I now have a patch that adds it to `PACKAGE_NAMES` and removes the `make` calls and the tests work correctly (added a failing to test to see that it was run, too). Tangentially, I kinda hate PACKAGE_NAMES and would love to get the `./...` alias working for all of this kind of stuff but I don't have a perfect solution for all the work it's doing, right now. 

I can post that patch as a follow-up. Would rather keep this one focused on the deduping.

Reply:
Yeah, separate patch is fine.

Reply:
Oh, just dug deeper, and it looks like maybe it was for the separated coverage calculation? https://github.com/mozilla-services/autograph/blob/6bfbe916169f2f79c19f72cc8f3a7b62155720d8/bin/run_unit_tests.sh#L25

I think it'll be fine if we combine it in, but I'll admit I've not use the goveralls stuff.